Core/Core/View/Base/LabelButton.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  LabelButton.swift
+//  Core
+//
+//  Created by Eugene Yatsenko on 10.10.2023.
+//
+
+import SwiftUI
+
+public struct LabelButton: View {
```

Comment:
Can you rename it to `SocailAuthButton` because we can't say we can generalize this button style for other parts of the app?